src/components/Chat/Chat.tsx

```diff
@@ -184,7 +184,7 @@ const ChatWithLLM: React.FC = () => {
           <FiRefreshCw className="text-white" /> {/* Icon */}
         </div>
       </div>
-      <div className="flex-1 overflow-auto p-4 pt-0 bg-transparent">
+      <div className="flex flex-col overflow-auto p-4 pt-0 bg-transparent h-full">
```

Comment:
These are just layout changes for the prompt templates to be situated near to the input text area when its added later

Reply:
Should note that in absence of the additional prompt template, nothing changes about the layout behavior
![image](https://github.com/reorproject/reor/assets/58682208/2deaaedd-9857-4c0a-b6cd-1835a9f6a4a8)